en/sql/schema/table_stmt.rst

```diff
@@ -168,19 +168,23 @@ An attribute in a table can be created with an initial **SHARED** or **DEFAULT**
 
 The pseudocolumn allows for the **DEFAULT** value as follows.
 
-+-------------------+---------------+
-| DEFAULT Value     | Data Type     |
-+===================+===============+
-| SYS_TIMESTAMP     | TIMESTAMP     |
-+-------------------+---------------+
-| SYS_DATETIME      | DATETIME      |
-+-------------------+---------------+
-| SYS_DATE          | DATE          |
-+-------------------+---------------+
-| SYS_TIME          | TIME          |
-+-------------------+---------------+
-| USER, USER()      | STRING        |
-+-------------------+---------------+
++-------------------------------+---------------+
+| DEFAULT Value                 | Data Type     |
```

Comment:
please also add the missing ones:
* UNIX_TIMESTAMP()
* CURRENT_DATETIME
* CURRENT_TIMESTAMP
* CURRENT_TIME
* CURRENT_DATE